### ISSUE TITLE
ajax error event?

### ISSUE THREAD

**Issue by doodi-v1** [rstaib/jquery-bootgrid#345]
When you specify ajax:true, is there a way to also specify an error event? I've been unsuccessful finding this in the doc and I've tried adding the event to the bootgrid parameter directly and through the ajaxSettings to no avail. Example, in case my question is vague or incoherent:

	$(mainGridID).bootgrid({
		ajax: true,
		ajaxSettings: {
			error: function(jqXHR,textStatus,errorThrown){
				if(jqXHR.status===400){
					document.location.reload();
				}else{
					console.log(jqXHR,textStatus,errorThrown);
					$(".notice").html("Error: "+errorThrown );
				}
			}
		},
		error: function(jqXHR,textStatus,errorThrown){
			if(jqXHR.status===400){
				document.location.reload();
			}else{
				console.log(jqXHR,textStatus,errorThrown);
				$(".notice").html("Error: "+errorThrown );
			}
		},...

**Comment by derrekbertrand** [comment]
Yeah. I can't find this in the docs either, and the docs say you can't override `error`.
